components/plugin_marketplace/marketplace_modal.test.tsx

```diff
@@ -131,28 +146,37 @@ describe('components/marketplace/', () => {
 
         test('should fetch plugins when plugin status is changed', () => {
             const fetchPlugins = baseProps.actions.fetchPlugins;
-            const wrapper = shallow(<MarketplaceModal {...baseProps}/>);
+            const wrapper = shallow<MarketplaceModal>(<MarketplaceModal {...baseProps}/>);
 
             expect(fetchPlugins).toBeCalledTimes(1);
             wrapper.setProps({...baseProps});
             expect(fetchPlugins).toBeCalledTimes(1);
 
-            wrapper.setProps({...baseProps, pluginStatuses: {test: 'test'}});
+            const status: PluginStatusRedux = {
+                id: 'test',
+                name: 'test',
+                description: 'some description',
+                version: '1.0.0',
+                active: true,
+                state: 2,
+                instances: [],
+            };
```

Comment:
Are all these fields necessary for the test? One solution to not need to write all the fields is to cast the object to the type. If it is a partial representation, it should work.
e.g. `const status = {} as PluginStatusRedux;`

Reply:
Good point. I was't sure if it's more idiomatic to satisfy the full API or use the minimum required data. Simplified in 600057aa63e9dd3aae4a3afa503564159157ca27.